packages/mdc-textfield/README.md

```diff
@@ -194,10 +194,21 @@ CSS Class | Description
 
 ### Sass Mixins
 
+To customize the colors of any part of the text-field, use the following mixins. We recommend you apply 
+these mixins within CSS selectors like .foo-text-field:not(.mdc-text-field--focused) to select your inactive text-fields, 
```

Comment:
to select your unfocused text fields

(no dashes in text field, it is two words)